org.eclipse.lemminx/src/test/java/org/eclipse/lemminx/services/extensions/HTMLHoverExtensionsTest.java

```diff
@@ -39,6 +39,9 @@ public void testSingle() throws BadLocationException {
 		assertHover("<html></htm|l>", "</html>", 8);
 		assertHover("<html></html|>", "</html>", 8);
 		assertHover("<html></html>|");
+		assertHover("<html>hover|Text</html>", "hoverText", 6);
+		assertHover("<html>h|overText</html>", "hoverText", 6);
+		assertHover("<html> |</html>", " ", 6);

```

Comment:
Please add test with empty text

Reply:
When there's no text (eg.` assertHover("<html>|</html>")`) the `DOMNode` is considered to be a DOMElement (not `DOMText`). In other words, [this if statement](https://github.com/eclipse/lemminx/pull/621/files/353b25cec17c55eb9572786bb15968f3e3b86980#diff-b8239e61a89785e705c99fc4ee5fc4a1R66) evaluates to true. 

Since we are already doing a test for empty text [here](https://github.com/eclipse/lemminx/pull/621/files/353b25cec17c55eb9572786bb15968f3e3b86980#diff-a769ff934bc2fac796856d092dc07122R34), I don't know if there's a way to test `onText` hover with empty text.